src/components/Header/HamburgerBotton.tsx

```diff
@@ -0,0 +1,123 @@
+import {
+  SITEMAP,
+  phoneList,
+  registerAndLoginList,
+  userList,
+} from '@components/Header/menu';
+import {
+  Popover,
+  PopoverContent,
+  PopoverTrigger,
+} from '@components/ui/popover';
+import { Separator } from '@components/ui/separator';
+import useRWD from '@hook/useRWD';
+import { Menu, X } from 'lucide-react';
+import Image from 'next/image';
+import Link from 'next/link';
+import { useState } from 'react';
+
+type Props = {
+  isLoggedin: boolean;
+  onSignOut: () => void;
+};
+
+type List = {
+  name: string;
+  icon?: JSX.Element;
+  url: string;
+};
+
+const defaultAvatar = '/header__defaultAvatar.svg';
+const fakeAvatar = '/header__fakeAvatar.svg';
+
+const HamburgerBotton = ({ isLoggedin, onSignOut }: Props) => {
+  const [isOpen, setIsOpen] = useState(false);
+  const device = useRWD();
+
+  return (
+    <Popover open={isOpen} onOpenChange={setIsOpen}>
+      <PopoverTrigger className="mx-[3px] flex items-center justify-center rounded-full border border-primary p-3 data-[state=open]:mx-0 data-[state=open]:border-4">
+        {isOpen ? <X size={24} /> : <Menu size={24} />}
+        <Image
+          className="ml-2"
+          src={isLoggedin ? fakeAvatar : defaultAvatar}
+          alt="user"
+          width={40}
+          height={40}
+        />
+      </PopoverTrigger>
+      <PopoverContent className="h-0 w-0 border-none bg-transparent p-0">
+        <div className="no-scrollbar absolute right-[-68px] top-4 h-[100vh] w-[100vw] overflow-scroll bg-[#e8e4de] pt-6 xl:right-[-56px] xl:h-fit xl:max-h-[450px] xl:w-[272px] xl:rounded-[32px] xl:border-4 xl:border-black">
```

Comment:
remove top-4, spacing would be right
```suggestion
        <div className="no-scrollbar absolute right-[-68px] h-[100vh] w-[100vw] overflow-scroll bg-[#e8e4de] pt-6 xl:right-[-56px] xl:h-fit xl:max-h-[450px] xl:w-[272px] xl:rounded-[32px] xl:border-4 xl:border-black">
```